learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,96 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
+## Prerequisites
+
++ [docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
++ you [log in](https://docs.docker.com/engine/reference/commandline/login/) your docker with [docker hub]((https://hub.docker.com/)) account
++ you have and know your **public IP** address
++ you have **a bitcoin and a litecoin address** for receiving mining reward
++ you may need to setup your network to allow other bitmark nodes to communicate with your bitmark node. You can check more information on "Important Notice On Network" below.
```

Comment:
Please capitalize the first character of a bullet except for the special cases. In this section, I think the `you` should be `You`.

Reply:
I couldn't find the `"Important Notice On Network"`, could you link the text to the corresponding section? And do you mean `Important Network Notes`?

Reply:
- your environment is open for incoming connections through port 2136

And I see we try to guide readers to `Important Network Notes` more than once, and I guess readers are very likely to be forced to scroll down to the bottom for that section. Does it make sense that we won't guide the readers to `Important Network Notes` section until they find the node is not running properly?

And are there other common problems when running a node? Could `Important Network Notes` be a more general troubleshooting section?

Reply:
changed as request